### ISSUE TITLE
Split components page using shiny.router

### ISSUE THREAD

**Issue by filipstachura** [Appsilon/shiny.semantic#50]
Currently we use single page layout for components documentation. It would be great to use router and have separate page for buttons and icons, etc.

This way it would look more similar to native Semantic UI documentation, which is additional advantage.

**Comment by marekrogala** [comment]
good idea! How about going even further and implementing this using _semantic.dashboard_?

**Comment by filipstachura** [comment]
That would be great!

**Comment by marekrogala** [comment]
In this case - CC @maju116 you might want to take this use-case into account in your work :smile:
